examples/container-lambda-worker/package.json

```diff
@@ -11,18 +11,18 @@
     "cdk": "cdk"
   },
   "devDependencies": {
-    "@types/jest": "^29.5.4",
+    "@types/jest": "29.5.4",
     "@types/node": "20.5.9",
     "aws-cdk": "2.1018.0",
-    "jest": "^29.6.4",
-    "ts-jest": "^29.1.1",
-    "ts-node": "^10.9.1",
-    "typescript": "~5.2.2"
+    "jest": "29.6.4",
+    "ts-jest": "29.1.1",
+    "ts-node": "10.9.1",
+    "typescript": "5.2.2"
   },
   "dependencies": {
     "aws-cdk-lib": "2.200.1",
-    "cdk-ecr-deployment": "^2.5.30",
-    "constructs": "^10.0.0",
-    "source-map-support": "^0.5.21"
+    "cdk-ecr-deployment": "2.5.30",
+    "constructs": "10.0.0",
```

Comment:
This is quite a downgrade, from 10.4.2.
If we're pinning things like this, they should be pinned to where they were.